src/constants/exchangeConstants.js

```diff
@@ -41,7 +44,10 @@ export const SLOW = 'min';
 export const NORMAL = 'avg';
 export const FAST = 'max';
 
+// WBTC.Cafe
 export const ALLOWED_SLIPPAGE = 3; // 3%
+export const WBTC_PENDING_TRANSACTION = 'WBTC_PENDING_TRANSACTION';
+export const MIN_WBTC_CAFE_AMOUNT = 0.0011; // WBTC.Cafe won't work for lower amounts
```

Comment:
I don't know that. This isn't even officially stated - the service just won't work for any lower amount. While their web app doesn't handle it in any way, I thought it would make sense to display an error message so that the user knows why it's not working. 

I doubt it will go lower, because the fees would be higher than the transaction's amount. We might want to check if it went higher every now and then though.

Reply:
Let's leave it as it is for now - we can always patch this later in the event that something does change.